corehq/apps/cowin/repeaters.py

```diff
@@ -0,0 +1,73 @@
+from django.utils.translation import ugettext_lazy as _
+
+from corehq import toggles
+from corehq.apps.cowin.repeater_generators import (
+    BeneficiaryRegistrationPayloadGenerator,
+    BeneficiaryVaccinationPayloadGenerator,
+)
+from corehq.apps.hqcase.utils import update_case
+from corehq.motech.repeaters.models import CaseRepeater
+
+
+class BaseCOWINRepeater(CaseRepeater):
+    class Meta:
+        app_label = 'repeaters'
+
+    @classmethod
+    def available_for_domain(cls, domain):
+        return toggles.COWIN_INTEGRATION.enabled(domain)
+
+    def get_headers(self, repeat_record):
+        headers = super(BaseCOWINRepeater, self).get_headers(repeat_record)
```

Comment:
(This is fine as it is, but you can use Python 3 syntax for `super()`.)

Reply:
https://github.com/dimagi/commcare-hq/pull/29949/commits/74d454f278f8b633ef9dcb7b7e36ea6087ef5249